src/api/chartDataFeed.ts

```diff
@@ -1,8 +1,12 @@
+import {uniq} from 'rambda';
 import * as topics from '../api/topics';
 import {InstrumentModel, MarketType, PriceType} from '../models/index';
 import * as mappers from '../models/mappers/index';
+import {DateSplitter} from '../utils/index';
 import {PriceApi} from './index';
 
+const splitter = DateSplitter();
```

Comment:
there is no need to mimic function as a class

you can import fns like this `import * as fns from 'module'`